mongo/options/gridfsoptions.go

```diff
@@ -225,6 +225,10 @@ type GridFSFindOptions struct {
 
 	// The maximum amount of time that the query can run on the server. The default value is nil, meaning that there
 	// is no time limit for query execution.
+	//
+	// Deprecated: This option is deprecated and will eventually be removed in version 2.0 of the driver. The more general
+	// Timeout option should be used in its place to control the amount of time that GridFS operations can run before
+	// returning an error. MaxTime is still usable through the deprecated setter.
```

Comment:
@kevinAlbs , nope `GridFSFindOptions.MaxTime` should also be deprecated; good catch. As part of GODRIVER-2346, I'll pass the `Timeout` from the `Client` down to GridFS operations to be used as both the `readDeadline` and `writeDeadline` if none is specified by the user.